### PR TITLE
Feat: gridsub

### PR DESCRIPTION
**Pull request by DiegoCDO**

### CODE REVIEW

File: bin/gridsub.py [hunk 1]
```diff
@@ -66,22 +74,99 @@ class JSON(str):
...
+    for job in jobs:
+        response = submit_campaign(job, client, campaign_id=campaign)
+        body = response.read()
+
+        if response.status == 201:
+            print(f"Jobs successfully added to campaign {campaign}")
+            if details:
+                print(json.dumps(json.loads(body.decode()), indent=2))
+        else:
+            raise RuntimeError(f"Error submitting campaign: {body}")
```
Comment (KillianGricourt) [inline bin/gridsub.py:169]: submitting campaign ? Shouldn't it be submitting job ?

**Review by KillianGricourt — COMMENTED** [review]
If comment solved, you can rebase

**Review by KillianGricourt — DISMISSED** [review]

**Comment by DiegoCDO** [comment]
I just fix the error message and rebase main into the branch

File: bin/gridsub.py (hunk 1 above)
Comment (DiegoCDO) [reply bin/gridsub.py]: I fixed the error message in the last commit

**Review by KillianGricourt — APPROVED** [review]
ok

**Pull request merged**